.github/workflows/periodic.yml

```diff
@@ -1,13 +1,18 @@
 ---
-name: Markdown Link Checks
+name: Periodic
+
 on:
   schedule:
     - cron: "0 0 * * 0"
+  push:
+    tags:
+      - 'v**'
+
```

Comment:
```suggestion
```

Otherwise you'll get "periodic" checks on each tag push (which would be confusing)

Reply:
I thought we agreed we wanted to run periodic checks on tag pushes, for example so we get the "all files" version of the MD links job, and also so we have all job logs for releases if we add additional periodic tests (like perf).

Reply:
Removed